client-s3/README.md

```diff
@@ -0,0 +1,22 @@
+# client-s3
+
+Backfila client backend implementation to backfill all s3 objects in a bucket that match a prefix. 
+You specify a bucket via `getBucket`. You specify the prefix either statically by `staticPrefix` or by 
+overriding `getPrefix`. Each s3 object is a separate partition. You must define a record strategy to 
+divide up the s3 object into batches.
+* Object names must be less than 45 characters after the prefix.
```

Comment:
where does this restriction come from?

Reply:
then lets make the restriction 300 bytes? I don't think anyone uses nonascii filenames in s3

Reply:
No strong feelings. I think 45 vs 75 was just an arbitrary buffer.

Reply:
I'm going to leave this the way it is for now. I'm happy to change it later if it makes sense.